keras_cv/layers/patch_encoder.py

```diff
@@ -0,0 +1,21 @@
+
+import tensorflow as tf
+from tensorflow.keras import layers
+
+"""
+Linear projection of patches made with `Patching` layer
+"""
+#@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class PatchEncoder(layers.Layer):
```

Comment:
Is this generally expected to be named "Embeddings" or "PositionEmbeddings" etc..?